Web3/Classes/Core/Json/EthereumBlockObject.swift

```diff
@@ -41,13 +41,13 @@ public struct EthereumBlockObject: Codable {
     public let receiptsRoot: EthereumData
 
     /// 20 Bytes - the address of the beneficiary to whom the mining rewards were given.
-    public let miner: EthereumAddress
+    public let miner: EthereumAddress?
```

Comment:
Are those fields optional now?

Reply:
They can be 'null' for pending logs. Without optional it can't be used for pending blocks